components/common/src/command/package/install.rs

```diff
@@ -185,6 +269,7 @@ pub fn start<P1, P2>(
     fs_root_path: P1,
     artifact_cache_path: P2,
     token: Option<&str>,
+    install_mode: &InstallMode,
```

Comment:
Is there a need for `install_mode` to be a reference rather than a `Copy` type? Since it's a small simple type, using a reference implies to me that it's modified somewhere and we want those modifications to be shared by the various places references are held, but I don't see that that is the case.

Reply:
Yep, a copy would work just as well, but I was trying to keep the function arguments more consistent in that they are almost all immutable borrows. Currently the `enum` is zero-sized, but at some future point it might contain something and I figured I may as well leave it in a borrow-first state.

I also would read this type in a function as a sign that I shouldn't try to change this value downstream, whereas with a copied value it's a little more open to interpretation.

But yes, the copy would work fine here as well.